bvinson23/finish-finder
# Bv boardform

# Description
User can create a board using the dropdown now.
## Type of change
# Testing Instructions
Go through and select one of each finish, name the board and click save. Check to make sure the board has been created.
# Checklist:
